src/main/java/com/rabbitmq/client/impl/AMQChannel.java

```diff
@@ -135,6 +137,20 @@ public AMQCommand exnWrappingRpc(Method m)
         }
     }
 
+    public CompletableFuture<Command> exnWrappingAsyncRpc(Method m)
```

Comment:
I now see we already have a "non-async" method with a similar name from way back. Disregard the above.